packages/pluggableWidgets/barcode-scanner-web/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "barcode-scanner-web",
+  "widgetName": "BarcodeScanner",
+  "version": "2.0.0",
+  "description": "Displays a progress bar",
+  "copyright": "Mendix Technology B.V.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendix/widgets-resources.git"
+  },
+  "config": {
+    "mendixHost": "http://localhost:8080",
+    "developmentPort": 3000
+  },
+  "packagePath": "com.mendix.widget.web",
+  "scripts": {
+    "start": "cross-env MPKOUTPUT=BarcodeScanner.mpk pluggable-widgets-tools start:server",
```

Comment:
This is a new widget, so we don't have to deal with legacy mpk file name conventions, like with the progress bar widget. Therefore, you can remove the `MPKOUTPUT` environment variable.

Reply:
Same for the scripts below.